server/src/handlers/chunk_handler.rs

```diff
@@ -45,7 +45,7 @@ use utoipa::ToSchema;
     "weight": 0.5,
     "split_avg": false
 }))]
-pub struct CreateChunkData {
+pub struct ChunkData {
```

Comment:
I much prefer `CreateChunkData` to this name. Nit, but this makes it seem like this is the `model` instead of the Create request payload.

Reply:
Its kinda weird, this *is* the model now the CreateRequestPayload is the enum which can be one or the other

Reply:
Maybe call it `CreateSingleChunkData`? 

Or `CreateChunkData` and `CreateChunksData`?

Reply:
This is exposed via the generated API clients

Reply:
Modified it, slighlty nastier code but better docs